source/elements/oneMKL/source/domains/data_fitting/terms.rst

```diff
@@ -0,0 +1,324 @@
+.. _terms:
+
+Common Terms
+============
+
+.. contents::
+    :local:
+    :depth: 2
+
+Glossary
+--------
+
+Assume we need to interpolate a function ``f(x)`` on the ``[a, b)`` interval using splines.
+
+Let's break ``[a, b)`` into sub-intervals by ``n`` points :math:`x_i`
+called partition points or simply **partition**.
```

Comment:
The language seems to be a little informal and not exactly clear to me what is what. Do you mean this:  

We define a **partition** of ``[a, b)`` to be a collection of :math:`n` sub-intervals :math:`\left\{ \left[x_{i-1}, x_{i}\right) \vert i = 1, \dots, n\right\}` where  :math:`x_0 = a`, and :math:`x_n = b`.  The :math:`n+1` points, :math:`x_i`, are called the **partition points**.

Reply:
I guess reading below, your partition is not the subintervals, but the collection of points ?  IS that mathematically correct?  or is the partition really the set of sub-intervals adn the partition points or nodes are the end points of each sub--interval ?

Reply:
You're right partition means a collection of points in our terms.
Regarding math. It's a collection of points. I'm not sure if it's a good proof (sorry of not), but here it is: https://en.wikipedia.org/wiki/Partition_of_an_interval

What if we say following?
>We define a **partition** of `[a, b)` to be a collection of :math:`n` points
(called **partition points**) :math:`\left\{ x_{i}\right}, i = 1, \dots, n` such as
:math:`a = x_1 < x_2 < \dots < x_n = b`.

Reply:
that looks better